src/backend/app/config.py

```diff
@@ -104,6 +104,8 @@ def assemble_db_connection(cls, v: Optional[str], info: ValidationInfo) -> Any:
     EMAILS_FROM_EMAIL: Optional[EmailStr] = None
     EMAILS_FROM_NAME: Optional[str] = "Drone Tasking Manager"
 
+    NODE_ODM_INTERNAL_DOCKER_URL: Optional[str] = "http://odm-api:3000"
```

Comment:
This could just be NODE_ODM_URL, and will either be the internal url if on the same machine, or an external url if hosted externally 👍